### DISCUSSION TITLE
Pimd scoped not working .

### DISCUSSION THREAD

**Discussion by userbsd** [troglobit/pimd#210] [Q&A]
Hi , a few day ago i maded an experimental stend to work with multicast traffic. I use pimd for routing on freebsd 13. in pimd.conf i writed only one row - "phyint re0 scoped 238.0.0.0/8" but when i run pimd , the group 238.30.30.4 is forwarded anyway via my server. Can someone help me with this?How can i block the unlike mcast groups ?(except firewall tools)

**Comment by troglobit** [comment]
Pretty sure that code path hasn't been tested i quite some time.  We should add a test case for it.  Also, what version of pimd are you using?

**Comment by userbsd** [comment]
Hi Joachim , 
give me a few days to make more tests with different version of BSD and your PIMD application. i will made some schemes to show a couple of things that i do not understand ,anyway i builded a stend with 6 pc`s .

**Comment by userbsd** [comment]
![github1](https://user-images.githubusercontent.com/94917180/143908879-179ab6e3-f7be-4c6b-9cf5-7d439c86444f.png)

Experiment 1 : Try to get multicast via R2. 
Result: When R2 (192.168.16.1) becomes an RP for 238.40.8.23 the OS is crash and go to reboot. It happening only when R1 start to delivering a multicast traffic to R2. I tryed to replace the machines but without succes. , it still go to reboot.
